frontend/src/components/ShootWorkers/MachineImage.vue

```diff
@@ -136,6 +148,13 @@ export default {
           severity: 'warning'
         })
       }
+      if (this.notInList) {
+        hints.push({
+          type: 'text',
+          hint: 'This image may not be supported by the selected machine type',
+          severity: 'warning'
+        })
+      }
```

Comment:
Why do we return something in this setter?

Reply:
what do you mean - where do we return something in the setter?

Reply:
This was my fault. I thought this belongs to the setter of machineImage.